crates/vm/levm/src/vm.rs

```diff
@@ -110,6 +126,9 @@ impl VM {
                     accrued_substate: Substate::default(),
                     cache,
                     tx_kind: to,
+                    // FIXME
```

Comment:
What would be the thing that needs to be fixed here? I guess adding the accounts from the upper layer, but a comment or an issue would be better to make sure someone fixes it in the future.

Reply:
Oh, it is a leftover, I'll remove it asap.

Reply:
Done in [729e035](https://github.com/lambdaclass/ethrex/pull/1314/commits/729e035a913b1679ea46a587dc6c5dd587271e7b).